.github/workflows/prerelease-npm-version.yml

```diff
@@ -0,0 +1,57 @@
+# This workflow is used to manually bump a new prerelease version and publish to NPM.
+# The prerelease-npm-version action in this workflow is called on each main branch push with the "rc" preid and tag.
+
+name: Prerelease NPM Version
+
+on:
+  workflow_dispatch:
+    inputs:
+      version-type:
+        description: "The type of version bump to make. One of: major, minor, patch, premajor, preminor, prepatch, or prerelease."
+        type: string
```

Comment:
this is a nitpick, but this might benefit from being a `choice` type with defined options, to prevent mishaps from user input